llama_stack/providers/inline/post_training/torchtune/post_training.py

```diff
@@ -123,7 +123,8 @@ async def preference_optimize(
         training_config: TrainingConfig,
         hyperparam_search_config: dict[str, Any],
         logger_config: dict[str, Any],
-    ) -> PostTrainingJob: ...
+    ) -> PostTrainingJob:
+        raise NotImplementedError("preference_optimize is not yet implemented")
```

Comment:
```suggestion
        raise NotImplementedError()
```

I believe the exception will be self-explanatory